docs/current_docs/configuration/cloud.mdx

```diff
@@ -203,6 +203,17 @@ dagger -m github.com/shykes/daggerverse/hello@v0.3.0 call hello
 
 Trigger your CI workflow by pushing a commit or opening a pull request.
 
+### Public traces
+
+Dagger Cloud automatically detects if the traces your pipelines produce are coming
```

Comment:
I'd change this section to:

Dagger Cloud automatically detects if the traces produced by your pipelines originate from a CI system and a public repository. When both conditions are met, Dagger Cloud allows public access to these traces without requiring an invitation to the organization.

To modify this default setting, navigate to the organization settings page. Under the `Configuration` tab, toggle the `Trace Repository Visibility` switch to control access to trace data as needed.

Reply:
+1